map/src/manager/track/SaveTrackManager.js

```diff
@@ -166,6 +155,43 @@ export function saveTrackToLocalStorage({ ctx, track }) {
     });
 }
 
+export async function saveEmptyTrack(folderName, ctx) {
+    //create empty file
+    const convertedData = new TextEncoder().encode('');
+    const zippedResult = require('pako').gzip(convertedData, { to: 'Uint8Array' });
+    const convertedZipped = zippedResult.buffer;
+    const oMyBlob = new Blob([convertedZipped], { type: 'gpx' });
```

Comment:
Is it really required to use empty gzip file here? Is the upload-file API requires this?